cms/static/sass/elements/_modal-window.scss

```diff
@@ -297,6 +297,11 @@
     .ui-loading {
       box-shadow: none;
     }
+
+    .modal-actions .action-move.is-disabled {
+      border: 1px solid $gray-l1 !important;
+      background: $gray-l1 !important;
+    }
```

Comment:
Yes, but modal disabled was a bit different than the normal studio button disabled, so on review I had to change this.